packages/modelviewer.dev/data/docs.json

```diff
@@ -1023,6 +1028,16 @@
         "name": "pause()",
         "htmlName": "pause",
         "description": "Causes animations to be paused. If you want to reset the current animation to the beginning, you should also set the <span class='attribute'>currentTime</span> property to 0."
+      },
+      {
+        "name": "appendAnimation(animationName, options: {repetitions, pingpong, weight, timeScale, fade, warp})",
+        "htmlName": "appendAnimation",
+        "description": "Causes an animation to run simultaneously with and blend into the current animation. If you want to get the appended animations, you can use <span class='attribute'>appendedAnimations</span> property. You can specify the number of repetitions of the animation by setting the number of <i>repetitions</i> to any value greater than 0 (defaults to Infinity). Also if you set <i>pingpong</i> to true, alternately playing forward and backward (defaults to false). <i>weight</i> The degree of influence of this animation (in the interval [0, 1]), Values between 0 (no impact) and 1 (full impact) can be used to blend between several actions (defaults to 1). <i>timeScale</i> option can be used to slow down, speed up or play backwards animations. For example, if you set it to 0.5, the animation will play at half speed (defaults to 1). The <i>fade</i> option causes the weight of this animation to gradually increase from 0 to 1, The value of this option can be a boolean or float greater than 0 (in seconds), If the value is true, default duration (1.25 second) is applied (defaults to false). The <i>warp</i> option causes change playback speed, within the passed time interval, by modifying timeScale gradually from 0 to current timeScale value, the value of this option can be a boolean or float greater than 0 (in seconds), If the value is true, default duration (1.25 second) is applied (defaults to false)."
```

Comment:
Thanks for the detailed docs! Since this is a fairly complex feature, can you also add a few automated tests and an example? We use our examples as our manual testing plan, and I think it will help users quite a bit to see this API in action.